client/modules/IDE/components/CopyableInput.jsx

```diff
@@ -42,7 +43,12 @@ class CopyableInput extends React.Component {
           onMouseLeave={this.onMouseLeaveHandler}
         >
           <label className="copyable-input__label" htmlFor={`copyable-input__value-${label}`}>
-            {label}
+            <div style={{ 'display': 'flex', 'justify-content': 'space-between' }}>
+              {label} {hasPreviewLink ? (
+                <a style={{ 'text-align': 'right' }} target="_blank" href={value}>
```

Comment:
same here—can you add this style to the scss?

Reply:
you could also make this react logic simpler:
```js
{{ hasPreviewLink && <a>...</a>}}
```

Reply:
Actually, the text-align attribute was not even necessary, so I simply removed it ^^